packages/mdc-menu/README.md

```diff
@@ -147,198 +122,101 @@ To disable a list item, set `aria-disabled` to `"true"`, and set `tabindex` to `
+`getIndexForEventTarget(target: EventTarget) => number` | Checks to see if the `target` of an event pertains to one of the menu items, and if so returns the index of that item. Returns -1 if the target is not one of the menu items. The same notice for `index` applies here as above.
+`notifySelected(evtData: {index: number}) => void` | Dispatches an event notifying listeners that a menu item has been selected. The function should accept an `evtData` parameter containing the an object with an `index` property representing the index of the selected item. Implementations may choose to supplement this data with additional data, such as the item itself.
+`notifyCancel() => void` | Dispatches an event notifying listeners that the menu has been closed with no selection made.
+`saveFocus() => void` | Stores the currently focused element on the document, for restoring with `restoreFocus`.
+`restoreFocus() => void` | Restores the previously saved focus state, by making the previously focused element the active focus again.
+`isFocused() => boolean` | Returns a boolean value indicating whether the root element of the simple menu is focused.
+`focus() => void` | Focuses the root element of the simple menu.
+`getFocusedItemIndex() => number` | Returns the index of the currently focused menu item (-1 if none).
+`focusItemAtIndex(index: number) => void` | Focuses the menu item with the provided index.
+`isRtl() => boolean` | Returns boolean indicating whether the current environment is RTL.
+`setTransformOrigin(value: string) => void` | Sets the transform origin for the menu element.
+`setPosition(position: { top: string, right: string, bottom: string, left: string }) => void` | Sets the position of the menu element.
+`setMaxHeight(value: string) => void` | Sets `max-height` style for the menu element.
+
+### `MDCSimpleMenuFoundation`
+
+Method Signature | Description
+--- | ---
+`setAnchorCorner(corder: Corner) => void` | Sets the corner that the menu will be anchored to. See [constants.js](https://github.com/material-components/material-components-web/blob/cc299230728ba5a994866ebd31aaaf1a0f4cc87f/packages/mdc-menu/simple/constants.js#L73)
+`setAnchorMargin(margin: AnchorMargin) => void` | Sets the distance from the anchor point that the menu should be shown. 
+`open({focusIndex: ?number}) => void` | Opens the menu. Optionally accepts an object with a `focusIndex` parameter to indicate which list item should receive focus when the menu is opened. 
+`close(evt: ?Event)` | Closes the menu. Optionally accepts the event to check if the target is disabled before closing the menu.  
+`isOpen() => boolean` | Returns a boolean indicating whether the menu is open. 
+
+### Events
+
+Event Name | Data | Description
+--- | --- | ---
+`MDCSimpleMenu:selected` | {detail: { item: HTMLElement, index: number } } | Used to indicate when an element has been selected. This event also includes the item selected and the list index of that item.
```

Comment:
Let's wrap the "Data" column values in single backticks and remove the spaces within the object literals.

E.g.:

```
`MDCSimpleMenu:selected` | `{detail: {item: HTMLElement, index: number}}` | Used to ...
```